tools/cldr-rdf/src/main/java/org/unicode/cldr/tool/GenerateSubdivisions.java

```diff
@@ -84,6 +84,7 @@ static final class SubdivisionInfo {
 
     public static void main(String[] args) throws IOException {
         CLDRConfig.getInstance().getSupplementalDataInfo();
+        String outputDirectoryName = CLDRPaths.GEN_DIRECTORY + "subdivision";
```

Comment:
The changes in this file are just to print out "Writing ..." to indicate what files are being created. Also, `FileUtilities.openUTF8Writer` should get a directory name plus a file name; it was getting part of the directory in the directory name and the other part in the file name

Reply:
These changes are unnecessary. There is an environment variable, SHOW_FILES, which if present causes the various utilities in FileUtilities to log when files are opened. That can be used with any of the tools, not just this one.

The real problem is that we don't document SHOW_FILES in the file or in https://cldr.unicode.org/